### ISSUE TITLE
Is allOf a possible keyword in schema ?

### ISSUE THREAD

**Issue by keldorus** [fastify/fast-json-stringify#64]
Hi,

I got a schema like that for a route on fastify.
```json
{
  "200": {
    "type": "object",
    "allOf": [
      {
        "type": "object",
        "required": [
          "name"
        ],
        "properties": {
          "name": {
            "type": "string"
          },
          "tag": {
            "type": "string"
          }
        }
      },
      {
        "required": [
          "id"
        ],
        "type": "object",
        "properties": {
          "id": {
            "type": "integer"
          }
        }
      }
    ]
  },
  "default": {
    "type": "object",
    "required": [
      "code",
      "message"
    ],
    "properties": {
      "code": {
        "type": "integer"
      },
      "message": {
        "type": "string"
      }
    }
  }
}
```

And sadly, my validation function for the 200 (`responseSchemaDef[statusCode]`) look like this : 
```js
$main(obj) {
      var json = '{'
      var addComma = false
  
      json += '}'
      return json
}
```

I'm guessing that `allOf` is not available, but it's a [JSON schema ](http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.7.1) official keywork.

Can you add it?

Same for `oneOf` ?

Regards

**Comment by mcollina** [comment]
Would you mind sending us a PR? We are pretty busy wrapping up 1.0.0 of fastify these days.

**Comment by keldorus** [comment]
I try something

**Comment by keldorus** [comment]
So can I close this one ? :)

**Comment by mcollina** [comment]
let's close this, it was released on 0.16.0.
